src/psyclone/tests/psyir/backend/fortran_test.py

```diff
@@ -2304,7 +2304,7 @@ def test_fw_directive_with_clause(fortran_reader, fortran_writer):
     schedule.addchild(parallel, 0)
     assert '''!$omp parallel default(shared), private(i)
   !$omp master
-  !$omp taskloop num_tasks(32) nogroup
+  !$omp taskloop num_tasks(32), nogroup
```

Comment:
I don't - clause node is now covered by the same tests I guess (since its in language_writer).

I added a test for directive with clause in c now as well.